doc/devel/development_setup.rst

```diff
@@ -127,6 +127,9 @@ The simplest way to do this is to use either Python's virtual environment
         <file folder location>\Scripts\activate.bat  # Windows cmd.exe
         <file folder location>\Scripts\Activate.ps1  # Windows PowerShell
 
+      On some systems, you may need to type ``python3`` instead of ``python``.
+      For the technical reasons for this note, please see `PEP-394 <https://peps.python.org/pep-0394>`_.
```

Comment:
```suggestion
      For the technical reasons, please see `PEP-394 <https://peps.python.org/pep-0394>`_.
```
Can be left out because it does not add information.